src/OpenTK/Math/Vector2.cs

```diff
@@ -695,6 +721,50 @@ public static void Transform(ref Vector2 vec, ref Quaternion quat, out Vector2 r
             result = new Vector2(v.X, v.Y);
         }
 
+        /// <summary>Transform a Position by the given Matrix rotation using right-handed notation.</summary>
```

Comment:
In this case, I believe it refers to the order of multiplicands, in that the vector is on the right side of the matrix. It should probably be clarified.